### ISSUE TITLE
Can't add a button to form

### ISSUE THREAD

**Issue by klapaudius** [Becklyn/OrderedFormBundle#4]
I had an InvalidArgumentException thrown with message :

The extended type specified for the service "0" does not match the actual extended type. Expected "Symfony\Component\Form\Extension\Core\Type\ButtonType", given "Becklyn\OrderedFormBundle\Form\Extension\FormType"

```php
public function buildForm(FormBuilderInterface $builder, array $options)
    {
        $builder
            ->add( 'submit', ButtonType::class , [
                'label' => 'layout.button',
            ] )
        ;
    }
```
Tried in Symfony 4.2.4, 4.2.5, 4.2.6 and 4.2.7
revert back to 1.0.2 fix the issue

**Comment by klapaudius** [comment]
My mistake.... I'm making a PR....

**Comment by klapaudius** [comment]
Done sorry for that. Could you release the new version ?

**Comment by keichinger** [comment]
No worries :) I've just released v1.1.1. It should be available on Packagist pretty soon.

Thank you for your contribution!
